### PR TITLE
Support GeoJSON specification for named crs.

### PR DESCRIPTION
**Pull request by alvinlindstam**
I wanted support for: http://geojson.org/geojson-spec.html#named-crs

Adds "crs_type" as an option to the serializer, with preserved default behaviour.

### CODE REVIEW

File: djgeojson/serializers.py [hunk 1]
```diff
@@ -79,10 +79,16 @@ def start_serialization(self):
 
     def get_crs(self):
         crs = {}
-        crs["type"] = "link"
+        type = self.options.pop('crs_type', None)
```
Comment (leplatrem) [inline djgeojson/serializers.py]: rename to ``crs_type`` since type is reserved keyword

File: djgeojson/tests.py [hunk 2]
```diff
@@ -163,11 +163,28 @@ def test_simplify(self):
...
 
+    def test_named_crs(self):
+        serializer = Serializer()
+        features = json.loads(serializer.serialize(
+            [{'geom': 'SRID=4326;POINT Z (1 2)'}],
+            crs_type="name"))
+        self.assertEqual(
+            features['crs'], {"type": "name", "properties": {"name": "EPSG:4326"}})
+
+    # todo: should misspelled name raise error instead?
```
Comment (leplatrem) [inline djgeojson/tests.py]: Yes, raising a ValueError would be safer, although not fundamental :)

**Comment by leplatrem** [comment]
Thanks for this contribution, with some tests \o/

**Comment by leplatrem** [comment]
It looks good to me.

Please wait for makina to merge and release this :)

File: djgeojson/tests.py [hunk 3]
```diff
@@ -163,11 +163,27 @@ def test_simplify(self):
     def test_force2d(self):
         serializer = Serializer()
         features2d = json.loads(serializer.serialize(
-            [{'geom': 'SRID=4326;POINT Z (1 2 3)'}],
+            [{'geom': 'SRID=4326;POINT Z (1 2)'}],
```
Comment (gutard) [inline djgeojson/tests.py]: Why do you change this test ?

Comment (sweco-sealvi) [inline djgeojson/tests.py]: Don't know. I shouldn't have. I'll push an update.

**Pull request merged**